102-CMW-Star-Cloud-Monitoring/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CMW - Cloud Monitoring Website</title>
+    <style>
+        * {
```

Comment:
Please create separate files for the CSS content. It's easier to maintain and you don't want to pollute your HTML code.